src/Uno.UWP/Security.Credentials/PasswordVault.Android.cs

```diff
@@ -160,5 +164,37 @@ public override void Write(byte[] buffer, int offset, int count)
 					=> _adapter.Write(buffer, offset, count);
 			}
 		}
+
+		/// <summary>
+		/// Persister for devices bellow Android level 23.
+		/// RSA/ECB/PKCS1Padding only is supported and is not considered secure.
+		/// </summary>
+		private sealed class UnSecureKeyStorePersister : FilePersister
```

Comment:
As finally you just thrown `NotSupported` in each method, you should rename this class to something like`NotSupportedPersister` (wondering also the needs to inherit from `FilePersister` instead of implementing `IPersister`).